packages/loader/container-loader/src/container.ts

```diff
@@ -609,6 +619,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                 },
             },
             this.mc.logger,
+            config.serializedContainerState?.clientId,
```

Comment:
I'd assume that Container.clientId needs to also be set in constructor.

Reply:
Container.clientId comes from connectionStateHandler